JCDispatch/src/main/java/com/openlattice/integrations/jcDispatch/lib/NameParsing.java

```diff
@@ -1,29 +1,25 @@
-package lib;
+package com.openlattice.integrations.jcDispatch.lib;
+
+import com.openlattice.shuttle.util.Parsers;
 
-import com.auth0.jwt.internal.org.apache.commons.lang3.text.WordUtils;
 import java.util.Arrays;
 import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 
-public class nameParsing {
+public class NameParsing {
 
-    public static String formatText( Object obj ) {
+    public static String addSpaceAfterCommaUpperCase( Object obj ) {
         if ( obj != null ) {
-            String name = obj.toString().toLowerCase().trim();
-            name = name.replace( ",", ", " );
-            if (name.contains( "-" )) {
-                name = name.replace( "-", "- " );
-                return WordUtils.capitalize( name.trim() ).replace( "-", "-" );
-            }
-            return WordUtils.capitalize( name.trim() );
+            String name = Parsers.getAsString( obj ).toLowerCase();
```

Comment:
Anytime you invoke `Parsers.getAsString()`, you still have to check the result for `null` before doing anything else. So this method would become:

```
public static String addSpaceAfterCommaUpperCase( Object obj ) {
  String name = Parsers.getAsString( obj );
  if ( name != null ) {
    return name.replace( ",", ", " ).toUpperCase();
  }
  return null;
}
```

Please update the other places in the code where you invoke `Parsers.getAsString()` to follow this pattern:

```
public static String helperMethod( Object obj ) {
  String str = Parsers.getAsString( obj );
  if ( str != null ) {
    // do something with the string
    ...
  }
  return null;
}
```